### PR TITLE
Change the xy parameter of Rectangle to cover imshow()'s vertical axi…

### PR DESCRIPTION
**Pull request by iwmq**
matplotlib.patches.Rectangle's first parameter, xy, is previously only documented for data plotting functions in which xy means left and bottom coordinates. However, in imshow(), the vertical axis's direction is inverted, and xy means left and top coordinates. See https://github.com/matplotlib/matplotlib/issues/15401.

In this PR, a fix is made to explain xy as starting coordinates so as to make it suitable for both cases.

### CODE REVIEW

File: lib/matplotlib/patches.py [hunk 1]
```diff
@@ -707,7 +707,7 @@ def __init__(self, xy, width, height, angle=0.0, **kwargs):
         Parameters
         ----------
         xy : (float, float)
-            The bottom and left rectangle coordinates
+            The starting coordinates. For most data plotting functions, it is (left, bottom). For imshow(), it is (left, top).
```
Comment (jklymak) [inline lib/matplotlib/patches.py]: This has to fit on a line <=79 characters long.  I'd just say "minimum x and y coordinate of rectangle corner"

Comment (iwmq) [reply lib/matplotlib/patches.py]: Sure, it will be clear to use "minimum x and y coordinate of rectangle corner". Thank you.

Comment (ImportanceOfBeingErnest) [reply lib/matplotlib/patches.py]: Minimum is only true if the the angle is zero and the width/height is positive.  
So "starting coordinates" would be fine, or maybe "anchor point" or so.

**Comment by iwmq** [comment]
Agreed. Thanks for all your comments.

**Comment by iwmq** [comment]
Everyone, I will close this PR and create a new one. Thanks for all your comment and they are very useful for me.

**Pull request closed without merging**